packages/mui-base/src/useSelect/useSelect.ts

```diff
@@ -459,7 +496,8 @@ function useSelect<OptionValue, Multiple extends boolean = false>(
     getHiddenInputProps,
     getListboxProps,
     getOptionMetadata,
-    listboxRef: mergedListRootRef,
+    listboxRef: handleListboxRef,
+    listboxRootRef: mergedListRootRef,
```

Comment:
I was wondering if I should add the `listboxRootRef` here. It's really useful for our tests because we can test that the button ref is linked to it on `activeDescendant` mode, and the listbox ref is linked to it on DOM mode.

Reply:
This could be pretty confusing - from the outside people will have hard time figuring out what's the difference between `listboxRef` and `listboxRootRef`. Plus, in some cases `handleListboxRef` is passed in to useList where it's used as a parameter to `useForkRef`. Exposing it directly (instead of useForkRef's result) could lead to unsynchronized refs and nasty bugs.

Reply:
Removed